epymodelingsuite/dispatcher/builder.py

```diff
@@ -381,13 +391,21 @@ def build_calibration(
             {"end_date": calibration.fitting_window.end_date, "projection": False, "epimodel": model}
         )
 
+        # Collect user-defined distance function
+        if isinstance(calibration.distance_function, UserDefinedFunction):
+            dist_func = import_module(
```

Comment:
Probably needs update in how we import Callable. Refer to the comment for post-hoc function.